.github/workflows/ci.yml

```diff
@@ -11,6 +11,7 @@ on:
 
 env:
   NODE_VERSION: 10.x
+  DOCKER_REPO: sheltertechsf
```

Comment:
Nit: I believe "repo" refers to the combination of the org and the specific repo name, which would be `sheltertechsf/askdarcel-web` in this case. Maybe this variable should be named `DOCKER_ORG` instead? Alternatively, you could make the value `sheltertechsf/askdarcel-web`.

Reply:
Will do